tests/Microsoft.DotNet.Framework.Docker.Tests/SdkOnlyImageTests.cs

```diff
@@ -15,36 +15,31 @@ namespace Microsoft.DotNet.Framework.Docker.Tests
     [Trait("Category", "sdk")]
     public class SdkOnlyImageTests : ImageTests
     {
-        private static readonly ImageDescriptor[] s_imageData = new ImageDescriptor[]
-        {
-            new ImageDescriptor { Version = "3.5", OsVariant = OsVersion.WSC_LTSC2016 },
-            new ImageDescriptor { Version = "3.5", OsVariant = OsVersion.WSC_LTSC2019 },
-            new ImageDescriptor { Version = "3.5", OsVariant = OsVersion.WSC_LTSC2022 },
-            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_LTSC2016 },
-            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_LTSC2019 },
-            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_LTSC2022 },
-            new ImageDescriptor { Version = "4.8.1", OsVariant = OsVersion.WSC_LTSC2022 },
-        };
-
         public SdkOnlyImageTests(ITestOutputHelper outputHelper)
             : base(outputHelper)
         {
         }
 
         protected override string ImageType => "sdk";
 
-        public static IEnumerable<object[]> GetImageData()
-        {
-            return ImageTestHelper.ApplyImageDataFilters(s_imageData);
-        }
+        public static IEnumerable<object[]> GetImageData() =>
+            ImageTestHelper.ApplyImageDataFilters(TestData.GetImageData(), "sdk");
+
+        private static bool IsSkippable(ImageDescriptor imageDescriptor) =>
```

Comment:
Would it make sense to express this with via logic in the ApplyImageDataFilters method?  Anytime the SDK version doesn't align with the runtime version a filter should be applied.  This would help cleanup the tests here  in the sense they wouldn't have to be declared skippable.

Reply:
Yeah, forgot to mention that in the initial comments. The need to use skipping is because XUnit doesn't allow a theory's MemberData method to return an empty set. So we can't filter out the member data; the test needs to skip the stuff that it doesn't want instead. Same applies to your other comment.

Reply:
Oh right, I remember encountering this before.  That said, I still see value in expressing this via logic versus a static list.  The value is in maintainability as versions reach EOL (ha ha).

```
imageDescriptor.Version != imageDescriptor.SdkVersion;
```